mutiny-wasm/src/lib.rs

```diff
@@ -262,7 +261,11 @@ impl MutinyWallet {
         let inner =
             mutiny_core::MutinyWallet::new(storage, config, Some(logger.session_id.clone()))
                 .await?;
-        Ok(MutinyWallet { mnemonic, inner })
+        Ok(MutinyWallet {
+            mnemonic,
+            inner,
+            subscription_expire_time: sub_expired_time,
```

Comment:
really? that's such a hack

Reply:
not sure a better place to put it, maybe the config?

Reply:
Honestly I was fine with the old way of doing it. Now I'm questioning if you've covered every edge case paul was doing in the frontend previously.

Reply:
Where do you update this when the user subscribes?

Reply:
Honestly I would rather revert the subscription changes at this point.